lib/elixir/lib/enum.ex

```diff
@@ -1436,6 +1436,20 @@ defmodule Enum do
   end
 
   @doc """
+  Invokes `fun` for each element in the collection passing that element and the
```

Comment:
Keep the first line of the documentation concise and short. It is used as a summary by ExDoc.